packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -84,11 +84,105 @@ const testDatas = {
   ...validatorTestData,
 };
 
+const filteredOperations = [
+  /* missing route */
+  /* #5694 */
+  "getSyncCommitteeRewards",
+  "getBlockRewards",
+  "getAttestationsRewards",
+  "getDepositSnapshot", // Won't fix for now, see https://github.com/ChainSafe/lodestar/issues/5697
+  "getBlindedBlock", // #5699
+  "getNextWithdrawals", // #5696
+  "getDebugForkChoice", // #5700
+  /* #6080 */
+  "getLightClientBootstrap",
+  "getLightClientUpdatesByRange",
+  "getLightClientFinalityUpdate",
+  "getLightClientOptimisticUpdate",
+  "getPoolBLSToExecutionChanges",
+  "submitPoolBLSToExecutionChange",
+
+  /* 
+   #6168
+   /query/syncing_status - must be integer
+   */
+  "getHealth",
```

Comment:
Isn't it sufficient to just ignore syncing_status query param?

Reply:
Good idea, fixed